moped-editor/src/auth/user.js

```diff
@@ -55,8 +55,12 @@ export const destroyLoggedInProfile = () =>
  * Parses the user database data from localStorage
  * @return {Object}
  */
-export const getSessionDatabaseData = () =>
-  JSON.parse(localStorage.getItem(atdSessionDatabaseDataKeyName));
+export const getSessionDatabaseData = () => {
+  // if (localStorage.getItem(atdSessionDatabaseDataKeyName) === "undefined") {
+  //   return null
+  // }
```

Comment:
not sure if i should remove this or uncomment it since we haven't definitively found the cause of why i couldnt log in...

Reply:
Hmm, I think this might be it actually. Totally cool if you delete it, not a mystery as to what we are trying to do. 🙏

Reply:
👍